packages/core/content-type-builder/admin/src/components/ComponentList/index.js

```diff
@@ -6,6 +6,7 @@
 /* eslint-disable import/no-cycle */
 import React from 'react';
 import { get } from 'lodash';
+import { Box } from '@strapi/parts';
```

Comment:
Not related to this specific line, but the file doesn't really reflect what the component is, WYT?

I would suggest something closer to `ComponentListCell` since this cannot be used outside a table anyways